kartothek/core/common_metadata.py

```diff
@@ -78,6 +77,35 @@ def _schema_compat(self):
 
             schema = schema.remove_metadata()
             md = {b"pandas": _dict_to_binary(pandas_metadata)}
+            # https://github.com/JDASoftwareGroup/kartothek/issues/259
+
+            if schema is not None:
```

Comment:
why can it be `None` here? 3 lines above we just call `schema = schema.remove_metadata()`. Can this return `None`?